tests/prepare/install/data/empty.fmf

```diff
@@ -0,0 +1,8 @@
+summary: Do nothing in prepare, CLI will provide inputs
+prepare: []
+
+adjust:
+  - when: distro == ubuntu
+    prepare+:
+      - how: shell
+        script: apt-get update
```

Comment:
I'd say it really should not be user responsibility to call `apt-get update`. I'd suggest to add it to the `apt ` installer code instead. Even if it would be called every time before the `apt install` command (for now, to simplify the implementation). To me this approach just does not seem to be right. But, that would be definitely for a separate pull request.

Reply:
True, it will end up in tmt code, consider this as a remnant of times when the borderline was not as clear as it is now (i.e. any time before the Tuesday meeting this week :)

Reply:
Ack.